File: 프로그래머스/0/120909. 제곱수 판별하기/제곱수 판별하기.cs
using System;

public class Solution 
{
    public int solution(int n) 
    {
        bool isSquare = false;
        for(int i=1; i<=1000; i++) {
            if(n == i * i) {
                isSquare = true;
            }
        }
        int answer = 0;
        return answer = isSquare ? 1 : 2;
    }
}

File: 프로그래머스/0/120909. 제곱수 판별하기/제곱수 판별하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) 
{
    int i = 1;
    bool isbool = false;
    while(n >= i * i)
    {
        if(n == i * i)
        {
            isbool = true;
            break;
        }
        i++;
    }
    return isbool ? 1 : 2;
}